apps/console/src/features/sms-providers/pages/sms-providers.tsx

```diff
@@ -0,0 +1,712 @@
+                    "notifications.deleteConfiguration.error.description"),
+                level: AlertLevels.ERROR,
+                message: t("extensions:develop.smsProviders." +
+                    "notifications.deleteConfiguration.error.message")
+            })
+        );
+    };
+
+    /**
+    * Renders the loading placeholder.
+    */
+    const renderLoadingPlaceholder = () => {
+        return (
+            <div data-componentid={ `${componentId}-form-loading` }>
+                {
+                    [ ...Array(3) ].map((key: number) => {
+                        return (
+                            <Placeholder key={ key }>
+                                <Placeholder.Line length="very short" />
+                                <div>
+                                    <Placeholder.Line length="long" />
+                                    <Placeholder.Line length="medium" />
+                                </div>
+                            </Placeholder>
+                        );
+                    })
+                }
+            </div>
+        );
+    };
+
+    const providerCards: SMSProviderCardInterface[] = [
+        { icon: getSMSProviderIcons().twilio, id: 1, key: "TwilioSMSProvider", name: "Twilio" },
+        { icon: getSMSProviderIcons().vonage, id: 2, key: "VonageSMSProvider", name: "Vonage" },
+        { icon: getSMSProviderIcons().custom, id: 3, key: "CustomSMSProvider", name: "Custom" }
+    ];
+    /**
+     * Resolves the page description.
+     */
+    const resolvePageDescription = (): ReactElement => {
+        return (
+            <div>
+                <div style={ { whiteSpace: "pre-line" } }>
+                    {
+                        t("extensions:develop.smsProviders.subHeading")
+                    }
+                    <DocumentationLink
+                        link={ getLink("develop.smsProviders.learnMore") }
+                    >
+                        { t("extensions:common.learnMore") }
+                    </DocumentationLink>
+                </div>
+            </div>
+        );
+    };
+
+    const handleBackButtonClick = () => {
+        history.push(`${ AppConstants.getDeveloperViewBasePath() }/notification-channels`);
+    };
+
+    return (
+        <PageLayout
+            title={ t("extensions:develop.smsProviders.heading") }
+            pageTitle={ t("extensions:develop.smsProviders.heading") }
+            description={ resolvePageDescription() }
+            bottomMargin={ false }
+            contentTopMargin={ false }
+            pageHeaderMaxWidth={ true }
+            backButton={ {
+                onClick: handleBackButtonClick,
+                text: "Go back to notification channels"
+            } }
+            data-componentid={ `${componentId}-form-layout` }
+        >
+            { isSMSProviderConfigFetchRequestLoading || isDeleting || isLoading ? (
+                renderLoadingPlaceholder()
+            ) : (
+                <>
```

Comment:
No need of react fragment `<>` if there's only one child inside. You can directly use it
```suggestion
```